README.md

```diff
@@ -54,6 +56,7 @@ Inspired by [awesome](https://github.com/sindresorhus/awesome), [awesome-dotnet]
 ### Individual Controls
 
 * [AvalonEdit](https://github.com/icsharpcode/AvalonEdit) - (FREE, OPEN SOURCE) AvalonEdit is the name of the WPF-based text editor in SharpDevelop 4.x.
+* [AvaloniaEdit](https://github.com/AvaloniaUI/AvaloniaEdit) - (FREE, OPEN SOURCE) AvaloniaEdit is the name of a fork of the orginal AvalonEdit with more features like Syntax highlighting using TextMate grammars and themes, Code folding, Code completion, Fully customizable and extensible, Line numeration, Display whitespaces EOLs and tabs, Line virtualization, Multi-caret edition, Intra-column adornments, Word wrapping, Scrolling below document, Hyperlinks.
```

Comment:
Unfortunately, AvaloniaEdit is not a WPF control, but rather a port for the Avalonia framework. Could you remove it? Thanks